### ISSUE TITLE
Donate by liberapay

### ISSUE THREAD

**Issue by PEPERSO** [minbrowser/min#1433]
**Can you describe the problem you're facing?**
Hi ! I would like to donate to the project, but if I donate in dollars, as suggested by the "sponsor" page, their will be too much fees for the conversion from euro to dollars.

**What solution would you like?**
I don't know if it is possible, or if it would resolve this problem, but Liberapay looks like a good, and adapted way to finance open source.

Maybe paypal woud do the trick, I don't know, but liberapay handles the recurrence of support, which indeed ensures security for devs

Thanks a lot for taking the time to develop Min, it's a beautiful piece of code :heart_eyes: :1st_place_medal:

**Comment by PalmerAL** [comment]
Thank you, I appreciate it!

How bad are the fees? The reason I ask is that Liberapay says "We currently support processing payments through Stripe and PayPal", and as far as I know, both of those charge currency-conversion fees as well. Regardless of the payment method, my bank account is in USD, and so there's going to be a conversion fee somewhere, but if we could find a way to lower it that would be nice.

**Comment by PEPERSO** [comment]
Yeah, I was expecting an issue like this... I'll try to see what can be done for the currency conversion, I'll let you know !
